app-modules/prospect/src/Filament/Pages/RecruitmentCrmDashboard.php

```diff
@@ -56,6 +56,8 @@ class RecruitmentCrmDashboard extends BaseDashboard
 
     protected static string $routePath = 'recruitment-crm-dashboard';
 
+    protected static ?string $navigationIcon = '';
```

Comment:
Does `null` work here? Or does it have to be an empty string to not default to something?

Reply:
I've tried with null as I mentioned in the call but it doesn't work, so I went with the empty string.